modules/sdk-coin-ethlike/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@bitgo/sdk-coin-ethlike",
+  "version": "1.0.0",
+  "description": "BitGo SDK coin library for EthLike coins",
+  "main": "./dist/src/index.js",
+  "types": "./dist/src/index.d.ts",
+  "scripts": {
+    "build": "yarn tsc --build --incremental --verbose .",
+    "fmt": "prettier --write .",
+    "check-fmt": "prettier --check .",
+    "clean": "rm -r ./dist",
+    "lint": "eslint --quiet .",
+    "prepare": "npm run build",
+    "test": "npm run coverage",
+    "coverage": "nyc -- npm run unit-test",
+    "unit-test": "mocha"
+  },
+  "author": "BitGo SDK Team <sdkteam@bitgo.com>",
+  "license": "MIT",
+  "engines": {
+    "node": ">=18 <21"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BitGo/BitGoJS.git",
+    "directory": "modules/sdk-coin-ethlike"
+  },
+  "lint-staged": {
+    "*.{js,ts}": [
+      "yarn prettier --write",
+      "yarn eslint --fix"
+    ]
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "nyc": {
+    "extension": [
+      ".ts"
+    ]
+  },
+  "dependencies": {
+    "@bitgo/abstract-eth": "^21.5.1",
+    "@bitgo/sdk-core": "^26.15.0",
+    "@bitgo/statics": "^48.15.0",
+    "@ethereumjs/common": "^2.6.5",
```

```suggestion
    "@ethereumjs/common": "2.6.5",
```
Please lock the version by removing the caret symbol for external dependencies.